jumper.template

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+### BEGIN INIT INFO
+# Provides:          jumperagent
+# Required-Start:    
+# Required-Stop:     
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Jumper Logging Agent
+# Description:       Start Jumper Logging Agent
+### END INIT INFO
+# Author: Roy Razon <roy@jumper.io>
+
+dir="/opt/jumper_logging_agent"
+cmd="/opt/jumper_logging_agent/venv/bin/python2.7 /opt/jumper_logging_agent/agent_main.py"
+user="jumperagent"
+
+name=`basename $0`
+pid_file="/var/run/$name.pid"
+stdout_log="/var/log/$name.log"
+stderr_log="/var/log/$name.err"
+
+get_pid() {
+    cat "$pid_file"
+}
+
+is_running() {
+    [ -f "$pid_file" ] && ps `get_pid` > /dev/null 2>&1
+}
+
+case "$1" in
+    start)
+    if is_running; then
+        echo "Already started"
+    else
+        echo "Starting $name"
+        cd "$dir"
+        if [ -z "$user" ]; then
+            sudo $cmd >> "$stdout_log" 2>> "$stderr_log" &
+        else
+            sudo -u "$user" $cmd >> "$stdout_log" 2>> "$stderr_log" &
+        fi
+        echo $! > "$pid_file"
```

Comment:
What if the process crashed? Looks like `pid_file` still exists and `is_running` returns true. You can't start it then...